.github/workflows/wheels.yaml

```
@@ -28,11 +28,11 @@ jobs:
           - os: "ubuntu-20.04"
             os-name: "focal"
     env:
-      BUILD_COMMIT: 1.3.2
+      BUILD_COMMIT: maint-1.3
       PLAT: "${{ matrix.platform }}"
       MB_PYTHON_VERSION: "${{ matrix.python }}"
       MB_ML_VER: "2014"
-      BUILD_DEPENDS: "oldest-supported-numpy cython==0.29.28 wheel==0.34.2"
+      BUILD_DEPENDS: "oldest-supported-numpy cython>=0.29 wheel==0.34.2"
       TEST_DEPENDS: "oldest-supported-numpy attrs==21.4.0 pytest==6.2.5 click==8.0.3 mock boto3 packaging hypothesis==4.38.1 wheel==0.34.2"
```

Testing with the latest versions is best practice, since most errors and failures will be caught, especially on newer Python versions.
```suggestion
      TEST_DEPENDS: "oldest-supported-numpy attrs>=22.1 pytest>=7.1 click>=8.1 mock boto3 packaging hypothesis>=6.56 wheel>=0.37"
```